services/chrome-web-store/chrome-web-store-last-updated.service.js

```diff
@@ -0,0 +1,41 @@
+import { age } from '../color-formatters.js'
+import { formatDate } from '../text-formatters.js'
+import { NotFound, pathParams } from '../index.js'
+import BaseChromeWebStoreService from './chrome-web-store-base.js'
+
+export default class ChromeWebStoreLastUpdated extends BaseChromeWebStoreService {
+  static category = 'activity'
+  static route = { base: 'chrome-web-store/last-updated', pattern: ':storeId' }
+
+  static openApi = {
+    '/chrome-web-store/last-updated/{storeId}': {
+      get: {
+        summary: 'Chrome Web Store Last Updated',
+        parameters: pathParams({
+          name: 'storeId',
+          example: 'nccfelhkfpbnefflolffkclhenplhiab',
+        }),
+      },
+    },
+  }
+
+  static defaultBadgeData = {
+    label: 'extension last updated',
```

Comment:
I'd suggest changing this to `last updated`, which is shorter and more consistent with other activity badges.

Reply:
Sure, I've renamed it